stdlib/unittest/_log.pyi

```diff
@@ -0,0 +1,24 @@
+import logging
+import sys
+from types import TracebackType
+from typing import Generic, NamedTuple, Type, TypeVar
+from unittest.case import TestCase
+
+_L = TypeVar("_L", None, _LoggingWatcher)
+
+class _LoggingWatcher(NamedTuple):
+    records: list[logging.LogRecord]
+    output: list[str]
+
+class _AssertLogsContext(Generic[_L]):
+    LOGGING_FORMAT: str
+    records: list[logging.LogRecord]
+    output: list[str]
```

Comment:
I don't see these two attributes in the implementation in 3.9.

Reply:
I deleted them and added annotations for attributes that actually exist.